runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttOptionsConfigAdapter.java

```diff
@@ -45,6 +44,8 @@ public class MqttOptionsConfigAdapter implements OptionsConfigAdapterSpi, JsonbA
     private static final String AUTHORIZATION_CREDENTIALS_CONNECT_NAME = "connect";
     private static final String TOPICS_NAME = "topics";
     private static final String VERSIONS_NAME = "versions";
+    private static final String MQTT_VERSION_4 = "v3.1.1";
+    private static final String MQTT_VERSION_5 = "v5";
```

Comment:
Please rename the constants to better align with the updated values.
Perhaps have an `MqttVersion` enum with enum constants named for the specification versions, with an attribute on each for the corresponding protocol version to better document what's happening here via the code.